juan-pablo-lopez/Tarea 3/js/smallEnough.js

```diff
@@ -0,0 +1,13 @@
+function smallEnough (a, limit){
+    let numbers = a; 
+    let limite = limit;
+    let bool = false;
+    for(index of a){
```

Comment:
@juanpablo-lopez01 , use modern approaches for array operations like forEach, map, reduce.